src/site-specific-feature.js

```diff
@@ -31,7 +31,7 @@ export default class SiteSpecificFeature extends ConfigFeature {
     }
 
     /**
-     * @returns {HTMLFormElement|null}
+     * @returns {HTMLElement|null}
```

Comment:
This wasn't erroring anywhere, but it might start erroring in the future. In fact, I think typescript should have complained here, not sure why it wasn't. The selector could return any element, not just forms.